wdl/ExpansionHunter.wdl

```diff
@@ -278,3 +339,67 @@ task ConcatEHOutputs {
         maxRetries: select_first([runtime_attr.max_retries, runtime_default.max_retries])
     }
 }
+
+task RunReviewer {
+    input {
+        String sample_id
+        File realigned_bam
+        File realigned_bam_index
+        File vcf_gz
+        File reference_fasta
+        File? reference_fasta_index
+        File variant_catalog_json
+        String expansion_hunter_docker
+        RuntimeAttr? runtime_override
+    }
+
+    output {
+        File? reviewer_images_gz = "reviewer_images.tar.gz"
+        File? sample_metrics = "${sample_id}_metrics.csv"
+    }
+
+    command <<<
+        set -euxo pipefail
+
+        REF="$(basename "~{reference_fasta}")"
+        mv ~{reference_fasta} $REF
+        mv ~{reference_fasta_index} $REF.fai
+
+        gunzip -c ~{vcf_gz} > genotypes.vcf
+
+        python /opt/str/get_locus_metrics.py \
+            --sample-id ~{sample_id} \
+            --realigned-bam ~{realigned_bam} \
+            --reference $REF \
+            --genotypes genotypes.vcf \
+            --variant-catalog ~{variant_catalog_json} \
+            --output ~{sample_id}_metrics.csv
+
+        tar -czvf reviewer_images.tar.gz ~{sample_id}_*.svg
+    >>>
+
+    RuntimeAttr runtime_default = object {
+        cpu_cores: 1,
+        mem_gb: 4,
+        boot_disk_gb: 10,
+        preemptible_tries: 3,
+        max_retries: 1,
+        disk_gb: 20 +
+            (2 * ceil(
+                size(realigned_bam, "GiB") +
+                size(vcf_gz, "GiB") +
```

Comment:
I think your vcfs are small, but since you are decompressing it we usually use a decompression factor of 5-10 for vcfs, although the relationship is generally non-linear, particularly for multi-sample VCFs.

Reply:
That is a good point; I added the decompression factor `10` to be on the safer side.